shared/logutil/logutil.go

```diff
@@ -23,3 +27,24 @@ func ConfigurePersistentLogging(logFileName string) error {
 	logrus.Info("File logging initialized")
 	return nil
 }
+
+// LogDebugRequestInfoUnaryInterceptor this method logs the gRPC backend as well as request duration when the log level is set to debug
+// or higher.
+func LogDebugRequestInfoUnaryInterceptor(ctx context.Context, method string, req, reply interface{}, cc *grpc.ClientConn, invoker grpc.UnaryInvoker, opts ...grpc.CallOption) error {
```

Comment:
What is this?

Reply:
Grpc logging

Reply:
Could we add a bit more descriptive comments for this function or make the name simpler?

Reply:
changed the method name